src/output_lib/components/RespondToForm/Validators/NumberValidator.js

```diff
@@ -0,0 +1,7 @@
+function valid(number) {
+  return !/\D/.test(number);
```

Comment:
let's move this one out of the function and name it `re` as well just for consistency. I know it's trivial.